Week1/work.py

```diff
@@ -67,8 +82,21 @@ def even_sort(arr):
     '''
 
     ## Code Here
-    return None
-
+    
+    arr=array([15,2,6,88,7])
+    even_arr=([])
+    odd_arr=([])
+    for i in arr:
+        if i%2==0:
+            even_arr.append(i)
+        else:
+            odd_arr.append(i)
+        
+    even_arr.sort()
+    odd_arr.sort()
+    
+    ans=concatenate((even_arr,odd_arr))
```

Comment:
again do type casting for the array [like this](https://github.com/AMC-IITBHU/Assignments-2021/pull/18#issuecomment-766562023). And also consider changing to ```np.concatenate()```